lib/src/constants/constant_colors.dart

```diff
@@ -6,6 +6,7 @@ class AppColor {
   static const primaryBlack = Color(0xff030F09);
   static const primaryGrey = Color(0xff606060);
   static const secondaryGrey = Color(0xffA8A8A8);
+  static const red = Colors.red;
```

Comment:
We don't need this definition

Reply:
Sr mr.Khanh added it.I didn't check it.
![image](https://user-images.githubusercontent.com/35758292/132102050-06c4c81f-d2b4-4891-99bb-2a5045abbec7.png)